Resources/templates/CommonAdmin/ActionsAction/ActionsBuilderAction.php.twig

```diff
@@ -24,6 +24,11 @@ class ActionsController extends BaseController
      */
     public function objectAction(Request $request, $pk, $action)
     {
+        // this is needed for confirmed object actions
+        if ($action == 'edit' || $action == 'show') {
+            return $this->redirectToRoute('{{ builder.baseActionsRoute }}_'.$action, array('pk' => $pk));
+        }
+
```

Comment:
if you make edit or show confirmed and protected it does not work without this...

it is bug in current code as well... try it in the demo...